Dokument/Dateien/Kultur.tex

```diff
@@ -4,6 +4,54 @@ \subsection{Menschen}
+	
+\subsubsection{Frauen} 
+Frauen hingegen haben als Sammlerinnen eher die Fähigkeit zu beobachten und sich Dinge abzuschauen und zu verbessern. 
+Sie nutzen als Waffe eher ihren Verstand, da sie in der Muskelkraft meist unterlegen sind. 
+Außerdem ist Fingerfertigkeit meist eine Ihrer Stärken.
+Aufgrund ihrer Eigenschaften als Mutter wird Ihnen auch eher Empathie zugeschrieben. 
+Aus all diesen Gründen sind Frauen in beratenden Stellungen besonders gefragt.
+	
+\subsubsection{Zusammenspiel}
+Das allgemeine Bild in der Gesellschaft ist die Gleichstellung beide Geschlechter. 
+Das eine Geschlecht wäre ohne das jeweils andere nicht lebensfähig. 
+Es werden jeweils die Vorteile aus beiden gezogen, um das Bestmögliche aus der Gesellschaft herauszuholen.
+
+\subsection{Familie}
+Auch in der Familie zieht sich das ebenbürtige Bild von Frau und Mann durch. So werden die Aufgaben im Haushalt je nach den Stärken aufgeteilt (bspw. kann ein Mann vermutlich besser Wäsche waschen und eine Frau besser haushalten). 
+Die Aufteilung der Aufgaben ist dabei sogar notwendig, da beide gleichermaßen ins Arbeitsleben eingespannt sein können. 
+Es werden übrigens nur monogame Beziehungen geduldet.
+\\
+\\
+Ebenso verhält es sich mit der Kindererziehung. 
+Je nach den täglichen Tätigkeiten der Eltern können diese mitgenommen und entsprechend erzogen und aufgezogen werden. 
+So werden die beruflichen Fertigkeiten und allgemeinen Qualitäten direkt an die Kinder weitergegeben.
+
+Als Säuglinge werden die Kinder im Idealfall von noch lebenden, aber nicht mehr arbeitstüchtigen Großeltern betreut. 
+Jedoch nur solange, bis die Kinder in der Lage sind, ihre Eltern beim Tagesgeschäft zu begleiten.
+\\
+\\
+\textbf{Exkurs: Homosexualität}\\
+Homosexualität ist kein Verbrechen und wird auch nicht als unnatürlich angesehen. 
+Will ein homosexuelles Pärchen eine Lebensgemeinschaft eingehen, dann verpflichten sie sich damit automatisch Kinder aufzunehmen, die von Ihrer eigentlichen Familie nicht mehr versorgt werden können oder die keine Familie mehr haben.
+
+\subsection{Alter}
+Wie im vorhergehenden Punkt beschrieben, können ältere Menschen bei der Kinderbetreuung und -erziehung sehr hilfreich sein. 
+Aus diesem Grund sind sie recht gut angesehen. Außerdem können sie wertvolle Erfahrungen mit Jüngeren teilen. 
+Obwohl sie also keine Arbeit mehr leisten können, tragen sie positiv zur Gesellschaft bei.
+\\
+\\
+Es gibt jedoch einen Punkt, an dem dieses Bild kippt. 
+Werden die Menschen zu alt, dann könnte das einfach Volk denken, dass diese Menschen nie in ihrem Leben richtig arbeiten mussten und lediglich aus diesem Grund noch fit sind. 
+Oder aber sie hatten oder haben einflussreiche Unterstützer, die sie versorgen können. 
+Gerade vom einfachen Volk werden "zu" alte Menschen also eher skeptisch beäugt. 
+Außerdem tragen sie nicht mehr zum Funktionieren der Gesellschaft bei oder brauchen sogar zusätzliche Pflege, was das Bild nicht gerade besser erscheinen lässt.
+
```

Comment:
"Dem ist insbesondere so, weil die Armut in der Bevölkerung so stark ist und sie sehen müssen, wo sie bleiben. Verbunden hat sich das auch über die Religion mit der Ansicht, dass die Menschen die Gaben, die sie von den Göttern erhielten, zurückzahlen sollten, indem sie nach ihrem Ableben mit ihnen die bösen Götter bekämpfen."
oder so :D